challenges/deploy-loderunner/README.md

```diff
@@ -0,0 +1,56 @@
+# Deploy Loderunner
+
+## Background
```

Comment:
is it a "load test"?  Why do we deploy LR? What does that enable?

Reply:
Updated the caption to describe LodeRunner